src/playbacks/html5_audio/html5_audio.js

```diff
@@ -61,11 +61,11 @@ export default class HTML5Audio extends Playback {
   }
 
   getPlaybackType() {
-    return [0, undefined, Infinity].indexOf(this.el.duration) >= 0 ? 'live' : 'aod'
+    return [0, undefined, Infinity].indexOf(this.el.duration) >= 0 ? Playback.LIVE : Playback.AOD
   }
 
   stalled() {
-    if (this.getPlaybackType() === 'vod' && this.el.readyState < this.el.HAVE_FUTURE_DATA) {
+    if (this.getPlaybackType() === Playback.VOD && this.el.readyState < this.el.HAVE_FUTURE_DATA) {
```

Comment:
yes @flavioribeiro, I am working on that and I will send the pull request shortly